contributors/guide/non-code-contributions.md

```diff
@@ -18,14 +18,15 @@ Are you interested in any of the roles below? Come chat with us [on Slack](https
 These are roles that either span the project as a whole, or span several areas of the project. Most of the roles below can be considered "good-first-roles".
 
 - Community education
```

Comment:
Can we add in an item regarding the Community Meetings here, and add an item like "Show demos of interesting tech related to K8s" or similar?
https://github.com/kubernetes/community/blob/master/events/community-meeting.md

Reply:
done